<a href="https://colab.research.google.com/github/gibsonea/Biostats/blob/main/HomeworkSolutions/Solutions_BIOE3060Homework5.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Homework 5

---

# <a name="HW9q1">Question 1</a>

------------------------------------------------------------------------

Suppose a student is working on a project using data on pulse rates collected from a random sample of 100 students at her university.
She finds a 95% confidence interval for the mean pulse rate to be from 65.5 to 71.8 beats per minute.
Explain (in complete sentences) why each of the statements below would be an improper interpretation of this interval.



## <a name="HW9q1a">Question 1a</a>

------------------------------------------------------------------------

I am 95% sure that all students will have pulse rates between 65.5 to 71.8 beats per minute.

### Solution to Question 1a

------------------------------------------------------------------------

Explain (in complete sentences) why the statement is an **improper interpretation** of this interval.

<br>

The confidence interval is not being interpreted to estimate the value of the  mean pulse rate of all students at the university (population). Instead, the interval is being used to describe the distribution of individual student pulse rates in the population rather than the mean puse rate of all students at the school.



<br>  
<br>  



## <a name="HW9q1b">Question 1b</a>

------------------------------------------------------------------------

95% of all students at this university have pulse rates between 65.5 and 71.8 beats per minute.

### Solution to Question 1b

------------------------------------------------------------------------

Explain (in complete sentences) why the statement is an **improper interpretation** of this interval.


This has two big issues.

1. The confidence interval is not being interpreted as an estimate for the mean pulse rate of all students at the university (the population parameter of interest). Instead, the confidence interval is being used to describe the distribution of individual pulse rates in the population.

2. The confidence level is interpreted completely wrong. The 95% should be associated with the success rate of the interval. The 95% has nothing do with finding an interval that contains 95% of all individual pulse rates.



<br>  
<br>  



## <a name="HW9q1c">Question 1c</a>

------------------------------------------------------------------------

95% of all random samples of 100 students have a mean pulse rate between 65.5 and 71.8 beats per minute.

### Solution to Question 1c

------------------------------------------------------------------------

Explain (in complete sentences) why the statement is an **improper interpretation** of this interval.


The confidence interval is not being interpreted to estimate the value of the  mean pulse rate of all students at the university (population). Instead, the interval is being used to predict how likely we are to randomly pick a sample with a certain sample statistic rather than the mean pulse rate of all students at the school.


<br>  
<br>  

# <a name="HW9q2">Question 2</a>

------------------------------------------------------------------------

For all high school seniors who took the SAT exam in 2018, the mean math SAT score was $515$ with a standard deviation of $\sigma = 116$.
From a random sample of $25$ students at your university, you find the average SAT math score of the sample is $538$.
You forgot to compute the standard deviation of your sample, so you assume the standard deviation of math SAT scores at your university is the same as the standard deviation of the population, $\sigma = 116$.



## <a name="HW9q2a">Question 2a</a>

------------------------------------------------------------------------

Compute a 95% confidence interval for $\mu$, the mean SAT math score at this university.

-   Show your work by replacing each of the <mark>??</mark> below with an appropriate value.

### Solution to Question 2a

------------------------------------------------------------------------

i.  An initial point estimate is $\mu \approx $<mark>538</mark>.

ii. The value of $z_{\alpha/2}$ for a 95% confidence level is <mark>1.96</mark>.

-   Round your answer to two decimal places when typing it above.

In [ ]:
# Show work by entering a formula (using qnorm)
z.star <- qnorm(0.975, 0, 1)
z.star

[1] 1.959964

iii. The standard error of the underlying sampling distribution is $\mbox{SE}(\bar{X}) =$ <mark>23.2</mark>.

-   Round your answer to two decimal places when typing it above.


In [ ]:
# Show work by entering a calculation
# that gives the standard error
se2 <- 116 / sqrt(25)
se2

[1] 23.2

iv. The margin of error (MoE) for a 95% confidence interval is <mark>45.47</mark>.

-   Round your answer to two decimal places when typing it above.

In [ ]:
# Show work by entering a calculation
# that gives the margin of error.
moe2 <- z.star * se2
moe2

[1] 45.47116

v.  A 95% confidence interval for the mean SAT math score of all students at the university is from <mark>492.5</mark> to <mark>583.5</mark>.

-   Round your answers to one decimal place when typing them above.

In [ ]:
lower2 <- 538 - moe2
upper2 <- 538 + moe2

lower2
upper2

[1] 492.5288

[1] 583.4712

## <a name="HW9q2b">Question 2b</a>

------------------------------------------------------------------------

Interpret the meaning of the confidence interval you found in [Question 2a](#HW9q2a) in practical terms to a person who does not come from a statistics background.

-   Be sure to keep in the mind the cautions we discussed in class regarding interpretations of confidence intervals.

Based on your interpretation, answer the question below. Explain how you determined your answer.


> Do you believe based on the data this school performs better than average?


### Solution to Question 2b

------------------------------------------------------------------------

**Write your interpretation below:**

<br>

There is a 95% chance the interval from $492.5$ to $583.5$ contains the mean math SAT score of all seniors at this high school.

<br>  
<br>  

Do you believe based on the data this school performs better than average? Explain.

<br>

No. The the national average math SAT score is 518 which is in the confidence interval. Since it is plausible that the mean math SAT score of all seniors at this high school, we cannot be certain this high school is better than average.


<br>  
<br>  

# <a name="HW9q3">Question 3</a>

------------------------------------------------------------------------

A researcher is interested in the sugar content of vanilla ice cream.
They obtain random samples of $n=20$ brands (each sample is a half cup serving of ice cream) and finds an average of $18.05$ grams of sugar with a standard deviation of $5$ grams.
Assuming the distribution of sugar content of all brands of vanilla ice cream is normally distributed, **find a 90% confidence interval for the mean amount of sugar in a half cup serving of vanilla ice cream.**

-   Show your work by replacing each of the <mark>??</mark> below with an appropriate value.

### Solution to Question 3

------------------------------------------------------------------------

i.  An initial point estimate is $\mu \approx $ <mark>18.05 grams of sugar</mark>.

ii. The value of $t_{\alpha/2}$ for a 90% confidence level is <mark>1.729</mark>.

-   Round your answer to three decimal places when typing it above.



In [ ]:
# Show work by entering a formula
t.star <- qt(0.95, 20 - 1)
t.star

[1] 1.729133

iii. The standard error of the underlying sampling distribution is $\mbox{SE}(\bar{X}) =$ <mark>1.118</mark>.

-   Round your answer to three decimal places when typing it above.


In [ ]:
# Show work by entering a calculation
# that gives the margin of error.
moe3 <- se3 * t.star
moe3

[1] 1.933229

iv. The margin of error (MoE) for a 90% confidence interval is <mark>1.933</mark>.

-   Round your answer to three decimal places when typing it above.

In [ ]:
# Show work by entering a calculation
# that gives the margin of error.
moe3 <- se3 * t.star
moe3

[1] 1.933229

v.   A 90% confidence interval for the mean amount of sugar in a half cup serving of vanilla ice cream is from <mark>16.117</mark> grams to <mark>19.983</mark> grams.

-   Round your answers to three decimal places when typing them above.

In [ ]:
lower3 <- 18.05 - moe3
upper3 <- 18.05 + moe3

lower3
upper3

[1] 16.11677

[1] 19.98323

# Question 4

------------------------------------------------------------------------

United Airlines Flight 179 is a nonstop flight from Boston to San Francisco.
During 2019 the mean length of time for all flights was $357.9$ minutes and population standard deviation was $20.18$ minutes.
You want to construct a 95% confidence interval for the mean length of time of the flight in 2023 based on a sample of flights in 2023.
**What is the minimum size of the sample needed in order to have a margin of error of two minutes or less?**

-   You may assume the standard deviation of the population of all flights in 2023 is equal to the standard deviation of the all flight lengths in 2019.

## Solution to Question 4

------------------------------------------------------------------------

Show supporting work/calculations in the space below.
Answer without any supporting work will not earn credit.


<br>  

We solve the following equation for $n$.

$$\mbox{MoE} = z_{\alpha/2} \cdot \frac{\sigma}{\sqrt{n}}  = 1.96 \cdot \frac{20.18}{\sqrt{n}} = 2$$

<br>  

From the code above, we see $n \approx 391.11$.  Since $n$ must be an integer, and we want to be sure we have a margin of error **at most** $2$ minutes, we would need a sample of size **at least** $\mathbf{392}$ **flights** (bumping up to the next integer as opposed to rounding down).


In [ ]:
# Here's an R code cell if you'd like to use it!
((1.96 * 20.18)/2)**2

[1] 391.106

# <a name="HW9q5">Question 5</a>

------------------------------------------------------------------------

The dataset `Girls2004` from  the `resampledata` package contains samples of the the birth weights of baby girls born in Wyoming and Alaska. Before answering the questions you will need to first run the code cells below to install and load the `resampledata` package.



In [ ]:
install.packages("resampledata")  # install package in active session

Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)



In [ ]:
library(resampledata)  # load package


Attaching package: ‘resampledata’


The following object is masked from ‘package:datasets’:

    Titanic




In [ ]:
summary(Girls2004)  # summary of data

       ID        State   MothersAge Smoker       Weight       Gestation    
 Min.   : 1.00   AK:40   15-19: 6   No :69   Min.   :2182   Min.   :37.00  
 1st Qu.:20.75   WY:40   20-24:29   Yes:11   1st Qu.:3076   1st Qu.:38.00  
 Median :40.50           25-29:22            Median :3331   Median :39.00  
 Mean   :40.50           30-34:15            Mean   :3362   Mean   :39.14  
 3rd Qu.:60.25           35-39: 6            3rd Qu.:3709   3rd Qu.:40.00  
 Max.   :80.00           40-44: 2            Max.   :4592   Max.   :42.00  

## <a name="HW9q5a">Question 5a</a>

------------------------------------------------------------------------

Complete the given R code cell below to R find a 99% confidence interval for the difference in the mean weights of girls born in these two states. The units of weight for this data is grams. Include units with your estimate.

### Solution to Question 5a

------------------------------------------------------------------------

<br>

In addition to R code and output from that code below, **be sure to type a sentence indicating your answer in the space below.**

-   Do not leave it for your reader to interpret the output.
-   Round your answers to two decimal places when typing them.

<br>  

There is a 99% chance the interval from $9.57$ grams to $607.33$ grams contains the difference in mean birth weight of all female newborns in Alaska and the mean birth weight of all female newborns in Wyoming.


<br>  
<br>  


In [ ]:
Weight <- Girls2004$Weight  # creates a vector birth weights

# Create a vector of Alaska birth weights
Weight.AK <-subset(Girls2004, select = Weight,
                   State == "AK", drop = TRUE)

# Create a vector of Wyoming birth weights
Weight.WY <-subset(Girls2004, select = Weight,
                   State == "WY", drop = TRUE)

# Compute confidence interval
t.test(Weight.AK, Weight.WY, conf.level = 0.99)$conf.int

[1]   9.566536 607.333464
attr(,"conf.level")
[1] 0.99

## <a name="HW9q5b">Question 5b</a>

------------------------------------------------------------------------

Based on your previous answer, do you believe that there is a difference in the mean weights of babies born in these two states.
**Explain why or why not.**

### Solution to Question 5b

------------------------------------------------------------------------

**Be sure to provide a supporting explanation!**

<br>  

Yes, it is plausible that the mean birth weights are not equal.

Since a difference of 0 is not inside the confidence interval, it is not plausible that $\mu_A - \mu_W = 0$. Since it is not plausible that $\mu_A = \mu_W$, we can equivalently say it is plausible that the mean birth weights are not equal.

<br>  


## <a name="HW9q5c">Question 5c</a>

------------------------------------------------------------------------

Construct an approximate 99% confidence interval by following these steps below.

-   Show your work by replacing each of the <mark>??</mark> below with an appropriate value.

### Solution to Question 5c

------------------------------------------------------------------------

i.  The mean of the weights of the Wyoming sample is <mark>3207.9</mark> grams. The standard deviation of the weights of the Wyoming sample is <mark>418.31</mark> grams.

-   Round your answers to two decimal places when typing them above.


In [ ]:
# Enter a formula for the mean of the Wyoming sample
x.bar.wy <- mean(Weight.WY)
x.bar.wy

# Enter a formula for the standard deviation of the Wyoming sample
s.wy <- sd(Weight.WY)
s.wy

[1] 3207.9

[1] 418.3184

iI.  The mean of the weights of the Alaska sample is <mark>3516.35</mark> grams. The standard deviation of the weights of the Alaska sample is <mark>578.83</mark> grams.

-   Round your answers to two decimal places when typing them above.


In [ ]:
# Enter a formula for the mean of the Alaska sample
x.bar.ak <- mean(Weight.AK)
x.bar.ak


# Enter a formula for the standard deviation of the Alaska sample
s.ak <- sd(Weight.AK)
s.ak

[1] 3516.35

[1] 578.8336


iii. If we use the degrees of freedom of the $t$ distribution equal to the minimum of $n_1-1$ and $n_2-1$, the value of $t_{\alpha/2}$ for a 99% confidence interval is <mark>2.708</mark>.

-   Round your answer to three decimal places when typing it above.


In [ ]:
n.ak <- length(Weight.AK)
n.wy <- length(Weight.WY)

n.ak
n.wy

[1] 40

[1] 40

In [ ]:
# Show work by entering a formula
t.star5 <- qt(0.995, 40 - 1)
t.star5

[1] 2.707913

iv. Using the previous answers in [Question 5c](#HW9q5c), a 99% confidence interval for the the difference in the mean weights of girls born in these two states is from <mark>2.67</mark> grams to <mark>614.23</mark> grams.

-   Round your answers to two decimal places when typing them above.


In [ ]:
# Feel free to use code cell for calculations

point.est5 <- x.bar.ak - x.bar.wy
se5 <- sqrt( s.ak^2 / n.ak + s.wy^2 / n.wy)
moe5 <- se5 * t.star5

lower5 <- point.est5 - moe5
upper5 <- point.est5 + moe5

lower5
upper5

[1] 2.671955

[1] 614.228

v.  How does your previous answer compare the to confidence interval you found in [Question 5a](#HW9q5a)? **Explain in a complete sentence or two.**


<br>  

- In part (a), we have a 99% confidence interval from 9.57 to 607.33 grams.
- In (iv) we have a 99% confidence interval from 2.67 to 614.23 grams.

We see that both intervals contain only positive numbers and do not contain 0, so both would lead us to the conclusion that there is a difference in the birth weights of all newborns in Alaska compare to Wyoming. The interval in part a is more narrow than the interval obtained in part (iv) **since part (a) uses a more accurate estimate for the degrees of freedom when finding the margin of error**.

<br>



# <a name="HW9q6">Question 6</a>

------------------------------------------------------------------------

The drug diflunisal is used to treat mild to moderate pain due to osteoarthritis (OA) and rheumatoid arthritis (RA). The ocular effects of diflunisal had not been considered until a study was conducted on its effect on intraocular pressure in glaucoma patients who were already receiving maximum therapy for glaucoma.  Suppose the change (mean ± sd ) in intraocular pressure after administration of diflunisal (follow-up – baseline) among 10 patients whose standard therapy was methazolamide and topical glaucoma medications was −1.6 ± 1.5 mm Hg. Assess the statistical significance of the results using a hypothesis test.

## <a name="HW9q6a">Question 6a</a>

------------------------------------------------------------------------

What is the hypothesis you are testing? Write out the equations for the null and alternative hypothesis and explain in a sentence.

### Solution to Question 6a

------------------------------------------------------------------------

<br>

$H_0: \mu = 0 $
<br>
$H_1: \mu \ne 0$

<br>

Explanation:

The null hypothesis is that diflunisal does not have an effect on intraocular pressure.

The alternate hypothesis is that diflunisal does effect intraocular pressure.
<br>  
<br>  

## <a name="HW9q6b">Question 6b</a>

------------------------------------------------------------------------

Perform the test and report a p-value.  What can you conclude from the results.

### Solution to Question 6b

------------------------------------------------------------------------

Use the code cell below to calculate your results and provide a written explanation of what you conclude.

<br>
We will use a t test with a test statistic:

$t = \frac{\bar{x} - 0}{s/\sqrt{n}} = \frac{\bar{x}}{s/\sqrt{n}} = \frac{−1.6}{1.5/\sqrt{10}} = -3.37$

The critical values are -2.262 and +2.262 (the 2.5th and 97.5th percentile of the $t_{9}$ distribution).

Because t = -3.37 is < -2.262, we reject the null hypothesis with a 5% significance level.

The p-value is:  $p = 2 \cdot P(t_{9}<-3.37)$.  

Using the cumulative distribution function for the $t_{9}$ distribution we find the p value to be **0.0083**.

We conclude that there is a highly significant difference in intraocular pressure with diflunisal.

<br>


In [ ]:
# code cell for Question 6b
#calculating the test statistic
-1.6/(1.5/sqrt(10))
#calculating the critical values
qt(0.025,9)
qt(0.975,9)
#calculating the p value
pt(-3.37,9)*2

[1] -3.373096

[1] -2.262157

[1] 2.262157

[1] 0.008256983

# <a name="HW9q7">Question 7</a>

------------------------------------------------------------------------

 Iron-deficiency anemia is an important nutritional health problem in the United States. A dietary assessment was performed on 51 boys 9 to 11 years of age whose families were below the poverty level. The mean daily iron intake among these boys was found to be 12.50 mg with standard deviation 4.75 mg. Suppose the mean daily iron intake among a large population of 9- to 11-year-old boys from all income strata is 14.44 mg. We want to test whether the mean iron intake among the low-income group is different from that of the general population.

## <a name="HW9q7a">Question 7a</a>

------------------------------------------------------------------------

State the hypotheses you would use to answer this question.  Write out the equations for the null and alternative hypothesis and explain in a sentence.

### Solution to Question 7a

------------------------------------------------------------------------

<br>

$H_0: \mu = \mu_{0}$
<br>
$H_1: \mu \ne \mu_{0}$

<br>

Explanation:

The null hypothesis is that there is no difference in the mean of daily iron intake for boys below the poverty level and boys in the general population.

The alternative hypothesis is that there is a difference in the mean of daily iron intake for those below poverty level and general population.

<br>  
<br>  

## <a name="HW9q7b">Question 7b</a>

------------------------------------------------------------------------

Perform the test and report a p-value.  What can you conclude from the results.

### Solution to Question 7b

------------------------------------------------------------------------

Use the code cell below to calculate your results and provide a written explanation of what you conclude.

<br>

We first calculate our test statistic:

$t = \frac{\bar{x} - \mu_{0}}{s/\sqrt{n}} = \frac{12.50-14.44}{4.75/\sqrt{51}} = -2.92$

The critical values are -2.009 and +2.009 (the 2.5th and 97.5th percentile of the $t_{50}$ distribution).

Because t = -2.92 is < -2.009, we reject the null hypothesis with a 5% significance level.

The p-value is:  $p = 2 \cdot P(t_{50}<-2.92)$.  

Using the cumulative distribution function for the $t_{50}$ distribution we find the p value to be **0.0053**.

We conclude that there is a highly significant difference in iron intake.



<br>

In [ ]:
#calculating the test statistic
t = (12.5-14.44)/(4.75/sqrt(51))
t
#calculating the critical values
qt(0.025,50)
qt(0.975,50)
#calculating the p value
pt(t,50)*2

[1] -2.91671

[1] -2.008559

[1] 2.008559

[1] 0.005285124